TRACONBoundaries.geojson

```diff
@@ -304,6 +304,8 @@
+    {"type":"Feature","properties":{"id":"MSK","prefix":["MSK_W"],"name":"Moscow Approach (Sector A1+A3)","label_lat":"55.5","label_lon":"35.2"},"geometry":{"type":"MultiPolygon","coordinates":[[[[37.71194416666667,57.107777222222225],[37.05277722222222,56.82083277777778],[37.05277722222222,56.82083277777778],[36.821110833333336,56.718333333333334],[36.821110833333336,56.718333333333334],[36.65222166666666,56.583333333333336],[36.65222166666666,56.583333333333336],[36.38333277777778,56.36166638888889],[36.38333277777778,56.36166638888889],[36.12444388888889,56.14916611111111],[36.12444388888889,56.14916611111111],[36.05777777777777,56.092777222222225],[36.05777777777777,56.092777222222225],[35.84666638888889,55.913888888888884],[35.84666638888889,55.913888888888884],[35.70583305555556,55.840833333333336],[35.70583305555556,55.840833333333336],[35.53861083333333,55.75277777777778],[35.53861083333333,55.75277777777778],[35.555277499999995,55.54222166666666],[35.555277499999995,55.54222166666666],[35.575,55.38944388888889],[35.575,55.38944388888889],[35.59055527777778,55.275555555555556],[35.59055527777778,55.275555555555556],[36.653953888888886,54.73218222222222],[36.653953888888886,54.73218222222222],[37.33138833333334,54.6],[37.33138833333334,54.6],[37.38,54.586666111111114],[37.38,54.586666111111114],[37.626110833333335,54.93666666666666],[37.626110833333335,54.93666666666666],[37.63888861111111,55.05361055555555],[37.63888861111111,55.05361055555555],[37.64805555555556,55.13583305555556],[37.64805555555556,55.13583305555556],[37.64944416666667,55.155],[37.64944416666667,55.155],[37.655,55.219166111111115],[37.655,55.219166111111115],[37.66111111111111,55.27416611111111],[37.66111111111111,55.27416611111111],[37.64472166666667,55.3752775],[37.64472166666667,55.3752775],[37.54666666666667,55.363888333333335],[37.54666666666667,55.363888333333335],[37.355555555555554,55.72749944444445],[37.355555555555554,55.72749944444445],[37.38333277777778,55.726665833333335],[37.38333277777778,55.726665833333335],[37.62083305555556,55.749722222222225],[37.62083305555556,55.749722222222225],[37.74138888888889,55.916666388888885],[37.74138888888889,55.916666388888885],[37.619999722222225,56.12638888888889],[37.619999722222225,56.12638888888889],[37.782777777777774,56.13611111111111],[37.782777777777774,56.13611111111111],[37.81527777777777,56.436388888888885],[37.81527777777777,56.436388888888885],[37.72138805555556,56.56388888888888],[37.72138805555556,56.56388888888888],[37.8575,56.82638861111111],[37.8575,56.82638861111111],[37.99527777777778,57.03777722222222],[37.99527777777778,57.03777722222222],[37.71194416666667,57.107777222222225]]]]}},
```

Comment:
The "name" field is by standard just supposed to be the name of the position. Is "Sector A1+A3" something common that pilots utilize when communicating with the controllers? If this is not something not known to pilots, it should not be displayed to them. Other examples for this are N90 in New York, where each sector are identified as "2K, 2J" etc. but the only thing that is displayed to pilots is: "New York Approach"